pkg/controller/expcertificates/trigger/trigger_controller_test.go

```diff
@@ -0,0 +1 @@
+package trigger
```

Comment:
Still TODO I guess?

Reply:
Yep, right now we have integration tests for `test/integration/certificates` for the overall controller behaviour. I think some basic unit testing here that 'mocks out' the 'policy chain' so we can test the core functionality does make sense, especially if we think that the bulk of the logic in the controller will be fairly 'static' (i.e. not require significant additional changes/work in future).

I'll see what I can do 😄